.maintain/node-template-release/Cargo.lock

```diff
@@ -0,0 +1,616 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+version = 3
+
+[[package]]
+name = "adler"
```

Comment:
not sure about the creation of this new file?

Reply:
Yeah, this was unintended. #9919